content/docs/quickstart/hasura.md

```diff
@@ -2,23 +2,43 @@
 title: Run a Hasura App
 ---
 
-Hasura is a cloud-based GraphQL provider for existing databases. This guide will cover zero-coding integration between Neon cloud service and Hasura Cloud. By the end of this guide, we will have a working HTTP API endpoint that uses GraphQL to query our serverless database and responds with a set of rows.
+### Introduction
```

Comment:
Header level is different from other steps, should be `###` -> `##`?

Reply:
@ololobus We have discussed with @stepashka on what styling guide we will refer to which will be the Digital Ocean Styling Guide.

Introductions and Conclusions will be `### `and Steps are referred to as `##`.

Please refer to https://www.digitalocean.com/community/tutorials/digitalocean-s-technical-writing-guidelines.